ietf/review/utils.py

```diff
@@ -588,10 +588,12 @@ def blocks(existing, request):
                    and existing.reviewassignment_set.filter(state_id__in=("assigned", "accepted")).exists()
                    and (not existing.requested_rev or existing.requested_rev == request.doc.rev))
         request_closed = existing.state_id not in ('requested','assigned')
+        # Is there a review request for this document already in system
+        requested = existing.state_id in ('requested')
```

Comment:
Would it get the same result in the cases you are testing if you added `and existing.type_id == request.type_id` here, or is it specifically existing, say, early assignments that cause problems when LC is requested?